apps/web/src/features/earn/components/VaultDepositTxDetails/index.tsx

```diff
@@ -0,0 +1,30 @@
+import type { VaultDepositTransactionInfo } from '@safe-global/store/gateway/AUTO_GENERATED/transactions'
+import VaultDepositConfirmation from '@/features/earn/components/VaultDepositConfirmation'
+import { Box } from '@mui/material'
+import FieldsGrid from '@/components/tx/FieldsGrid'
+import TokenAmount from '@/components/common/TokenAmount'
+import { formatPercentage } from '@safe-global/utils/utils/formatters'
+
+const VaultDepositTxDetails = ({ info }: { info: VaultDepositTransactionInfo }) => {
+  const totalNrr = (info.baseNrr + info.additionalRewardsNrr) / 100
+
+  return (
+    <Box
+      sx={{
+        pl: 1,
+        pr: 5,
+        display: 'flex',
+        flexDirection: 'column',
+        gap: 1,
+      }}
```

Comment:
The Box component takes sx props directly, you shouldn't use the object notation.

Reply:
Changed it to use inline notation